src/SynthGrid.cpp

```diff
@@ -569,6 +579,214 @@ bool SynthGrid::onEvent(SDL_Event& event)
 		{
 			switch (event.key.keysym.sym)
 			{
+
+				case SDLK_RIGHT:
```

Comment:
I think we should refactor these, lots of duplicated code. Only the direction changes so this could be something like:

```
case SDLK_LEFT:
case SDLK_RIGHT:
  int direction = event.key.keysym.sym == SDLK_LEFT ? -1 : 1;
...
  mSelectedModule += direction;
...
  ... -> onDial(direction)
...
```

And so on. And similarly to the up/down directions.

Reply:
I will get on with this asap.

I should have thought of that really but am not too familiar with using switch statements just yet.

Reply:
It might be easier to think this as a separate method that is just called in the case.

```
case SDLK_LEFT:
  moveStuff(-1);
  break;
case SDLK_RIGHT:
  moveStuff(1);
  break;
```